azurelinuxagent/common/conf.py

```diff
@@ -55,7 +55,7 @@ def get(self, key, default_val):
 
     def get_switch(self, key, default_val):
         val = self.values.get(key)
-        if val is not None and val.lower() == 'y':  # pylint: disable=R1705
+        if val is not None and val.lower() == 'y':
```

Comment:
Similar comment to the one I left above, though this time I wonder if this warning is just a symptom of this code being unnecessarily verbose/repetitive. 

Not sure it's worth trying to refactor this to a potentially cleaner structure (especially with a lack of native switch statements in python), but we can always just turn the `elif` into an `if`. I think that would make it a bit more readable, to me at least.

Thoughts?

Reply:
I think the normal python replacement for a switch statement is to use a dictionary (a `defaultdict` probably better matches this case if that is easily imported across all our supported versions of python), if we want to go that route.

Reply:
Same as my answer above.